language-support.md

```diff
@@ -103,8 +104,73 @@ Language support might be different if you are a {{site.data.keyword.Bluemix_ded
   </tbody>
 </table>
 
-&#42; Sentiment detection for non-English languages works best on social media based data, like tweets. Further accuracy improvements for non-English sentiment are planned in the near future.
+&#42; Sentiment detection for this language works best on social media based data, like tweets. Further accuracy improvements are planned in the near future.
 
```

Comment:
the noun stack is probably more difficult to parse than splitting out. As in "...this language works best on data that is based on social media, such as tweets."